addons/modules/functions/fnc_moduleUnpackStaticWeapon.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to unpack and assemble a static weapon from supported backpacks.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleUnpackStaticWeapon
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _gunner = attachedTo _logic;
+deleteVehicle _logic;
+
+if (isNull _gunner) exitWith {
+    [LSTRING(NoUnitSelected)] call EFUNC(common,showMessage);
+};
+
+if !(_gunner isKindOf "CAManBase") exitWith {
+    [LSTRING(OnlyInfantry)] call EFUNC(common,showMessage);
+};
+
+if !(alive _gunner) exitWith {
+    [LSTRING(OnlyAlive)] call EFUNC(common,showMessage);
+};
+
+if (isPlayer _gunner) exitWith {
+    ["str_a3_cfgvehicles_moduleremotecontrol_f_errorPlayer"] call EFUNC(common,showMessage);
+};
+
+private _fnc_getCompatibleBases = {
```

Comment:
Saw that there was an earlier comment on extracting to separate function/file. Can do that or leave here, both are fine.

Reply:
Will leave here for now.